packages/mobile/src/recipients/recipient.ts

```diff
@@ -93,7 +94,7 @@ export function contactsToRecipients(contacts: MinimalContact[], defaultCountryC
           }
           e164NumberToRecipients[parsedNumber.e164Number] = {
             kind: RecipientKind.Contact,
-            displayName: contact.displayName,
+            displayName: contact.displayName || i18n.t('sendFlow7:mobileNumber'),
```

Comment:
What if we just populate this field with the actual phone number? just a thought, not really sure what's better

Reply:
Just curious: how come the displayName can be null since the interface has it non optional?

Reply:
I guess it can happen when reading from the addressbook there are no guarantees. But weird we haven't seen the crash so far. Also I agree we should add a check in the nameCompare function as well, or maybe use the getDisplayName function we already have which seems to do some null checking.

Reply:
Showing `mobileNumber` is consistent with how we deal with this elsewhere. `contact.displayName` should always be a string but for whatever reason that's not happening.

Not sure why this is showing up now though. I tried to track it down to changes we made in 1.12.2 but couldn't see how it's connected.